src/DataDepsGenerators.jl

```diff
@@ -32,16 +32,14 @@ function find_metadata(repo, dataname, shortname)
 
     mainpage, url = mainpage_url(repo, dataname)
     fullname = data_fullname(repo, mainpage)
-    if shortname == nothing
-        shortname = reduce((s,r)->replace(s, r, ""), fullname, ['\\', '/', ':', '*', '?', '<', '>', '|'])
-    end
+    shortname = data_shortname(repo, shortname, fullname)
 
     Metadata(
         fullname,
         shortname,
         website(repo, url, mainpage),
         description(repo, mainpage),
-        author(repo, mainpage),
+        format_authors(author(repo, mainpage)),
```

Comment:
MetaData  should have a field `authors::Opt{Vector{String}}`.
So all the authors can be stored.
And the `format_authors` belong with the other formatting commands (like `format_date`),
in the part that generates the code